### PR TITLE
Support fp16 training

### PR DESCRIPTION
**Pull request by ly015**

### CODE REVIEW

File: configs/top_down/hrnet/README.md [hunk 1]
```diff
@@ -26,12 +26,15 @@
...
+
+<sup>1</sup> Please refer to [fp16/README.md](/configs/fp16/README.md) for the method we use for mixed precision training.
 
 #### Results on AIC val set with ground-truth bounding boxes
 
 | Arch  | Input Size | AP | AP<sup>50</sup> | AP<sup>75</sup> | AR | AR<sup>50</sup> | ckpt | log |
 | :-------------- | :-----------: | :------: | :------: | :------: | :------: | :------: |:------: |:------: |
-| [pose_hrnet_w32](/configs/top_down/hrnet/aic/hrnet_w32_aic_256x192.py) | 256x192 | 0.323 | 0.762 | 0.219 | 0.366 | 0.789 | [ckpt](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_aic_256x192-30a4e465_20200826.pth) | [log](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_aic_256x192_20200826.log.json) |
+| [pose_hrnet_w32](/configs/top_down/hrnet/aic/hrnet_w32_aic_256x192.py) | 256x192 | 0.323 | 0.762 | 0.219 | 0.366 | 0.789 | [ckpt](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_aic_256x192-30a4e465_20200826.pth) | [log](https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w32_coco_256x192_fp16_dynamic_20210430.log.json) |
```
Comment (jin-s13) [inline configs/top_down/hrnet/README.md:37]: Is this a mistake?

File: setup.cfg [hunk 2]
```diff
@@ -19,6 +19,6 @@ line_length = 79
 multi_line_output = 0
 known_standard_library = pkg_resources,setuptools
 known_first_party = mmpose
-known_third_party = cv2,h5py,json_tricks,matplotlib,mmcv,munkres,numpy,packaging,poseval,pytest,scipy,seaborn,spacepy,titlecase,torch,torchvision,xtcocotools
```
Comment (jin-s13) [inline setup.cfg:22]: Why packaging is removed?

File: configs/top_down/hrnet/README.md (hunk 1 above)
Comment (ly015) [reply configs/top_down/hrnet/README.md:37]: Yes. I will fix it.

File: setup.cfg (hunk 2 above)
Comment (ly015) [reply setup.cfg]: It is automatically removed because it's no longer used.

**Review by jin-s13 — APPROVED** [review]

**Pull request merged**